### PR TITLE
Update geospatial.md

### PR DESCRIPTION
**Pull request by sav-norem**
update geo tutorial to include hugo shortcodes for tabbed examples

### CODE REVIEW

**Comment by sav-norem** [comment]
@dwdougherty could you give this a check?

File: docs/data-types/geospatial.md [hunk 1]
```diff
@@ -19,28 +19,28 @@ See the [complete list of geospatial index commands](https://redis.io/commands/?
 
 ## Examples
 
-Suppose you're building a mobile app that lets you find all of the electric car charging stations closest to your current location.
+Suppose you're building a mobile app that lets you find all of the rentable bike stations closest to your current location.
```
Comment (dwdougherty) [inline docs/data-types/geospatial.md:22]:
```suggestion
Suppose you're building a mobile app that lets you find all of the bike rental stations closest to your current location.
```

**Review by dwdougherty — APPROVED** [review]
Apart from the one change, this looks good.

**Review by dwdougherty — APPROVED** [review]
I built this PR in my test environment and the changes look good.

**Review by dmaier-redislabs — APPROVED** [review]

**Pull request merged**